packages/terra-tabs/src/Tabs.jsx

```diff
@@ -58,11 +58,19 @@ const propTypes = {
    * Key of the pane that should be open initially.
    */
   defaultActiveKey: PropTypes.string,
+
+  /**
+   * The viewport the element will be responsive to. One of `window`, `parent` or `none`.
+   * Note: If the responsive viewport is set to `none`, then tabs never _completely_
+   * collapse into a menu.
+   */
+  responsiveTo: PropTypes.oneOf(['window', 'parent', 'none']),
 };
 
 const defaultProps = {
   tabFill: false,
   fill: false,
+  responsiveTo: DependentViewport.PARENT,
```

Comment:
```suggestion
  responsiveTo: 'parent',
```
This will look better in the generated props table.